README.md

```diff
@@ -87,7 +89,206 @@ worker: `celery -A AIPscan.Aggregator.tasks worker --loglevel=info`
 
 ![screencap4](screencaps/aipscan_scatterplot_report.png)
 
+##Production intallation
```

Comment:
You might need to update your rendering engine if you've been previewing these somewhere - the `##TEXT` won't render as a H2 where `## text` will. See also `##Conclusion` below.

Current MD rendering: https://github.com/artefactual-labs/AIPscan/blob/acdec56028720e6ef0d0ca2c3d1a6c051dfbde93/README.md